oracle.py

```diff
@@ -32,11 +33,12 @@ def update_legality(self, legal_cards):
 
     def update_database(self, new_version):
         self.database.execute('DELETE FROM version')
-        self.database.execute('DELETE FROM card')
```

Comment:
It doesn't do anything? Do we ever update the card table without first dropping everything?